js/models/dataset.js

```diff
@@ -23,9 +23,11 @@ export default class Dataset extends Model {
      * Create or update the given dataset.
      */
     save(on_error) {
-        var ep = this.id ? 'datasets.update_dataset' : 'datasets.create_dataset';
+        if (this.id) {
+            return this.update(this, this.on_fetched, on_error);
+        }
         this.loading = true;
-        this.$api(ep, {payload: this}, this.on_fetched, on_error);
+        this.$api('datasets.create_dataset', {payload: this}, this.on_fetched, on_error);
```

Comment:
Yes, I agree, but
> For the futur, it would be interesting to normalize the create/save/update methods on models.

This PR is the minimal bugfix (no signature change, only fix the bug where it is), refactoring all the models and change it in every place where it's used wasn't in the scope. I think it needs a dedicated PR on the dev branch.